.gitmodules

```diff
@@ -1,7 +1,7 @@
-# Copyright (c) Microsoft Corporation.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
-[submodule "vcpkg"]
-	path = vcpkg
-	url = https://github.com/microsoft/vcpkg.git
-	fetchRecurseSubmodules = false
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+[submodule "vcpkg"]
+    path = vcpkg

```

Comment:
This one is usually machine generated and I think we should leave it alone; otherwise the 'git submodule' command will be not very useful.

Reply:
I have added allow-list logic to `validate.cpp` and reverted this change to `.gitmodules`.